Sources/App/Views/PackageController/Builds/BuildShow+View.swift

```diff
@@ -134,3 +143,15 @@ private extension Build.Status {
         }
     }
 }
+
+
+private extension DateFormatter {
+    static var displayFormatter: DateFormatter {
```

Comment:
I've been trying to consolidate the date formatters we use in the front end into `Date+ext.swift` so that we standardise a bit on how we display dates. It's no problem that this is a new format as we rarely need the specific time of something, but I wonder if we could move this there so we have a standard "with time" formatter?

Reply:
I've kept this private because I don't think we want this date formatter to be used in general. It's printing the date in UTC and we should ideally have this represented in the client's locale. I'm not sure how to best do this so UTC is the best I could come up with to get this out for now.

Reply:
OK, lets add that as a comment then. I'll make a PR.